static/src/javascripts/bootstraps/commercial.js

```diff
@@ -3,13 +3,13 @@ import config from 'lib/config';
 import { catchErrorsWithContext } from 'lib/robust';
 import { markTime } from 'lib/user-timing';
 import reportError from 'lib/report-error';
-import highMerch from 'commercial/modules/high-merch';
-import { articleAsideAdvertsInit } from 'commercial/modules/article-aside-adverts';
-import { articleBodyAdvertsInit } from 'commercial/modules/article-body-adverts';
+import { initHighMerch } from 'commercial/modules/high-merch';
```

Comment:
If this naming convention is only there to avoid conflicts, it is the responsibility of the calling module to deal with these conflicts, instead of spreading that responsibility over to N modules. So
```js
import { init as initHighMerch }
```
That is similar in Scala
```scala
import com.gu.xyz.{ init => initHighMerch }
```
👼

Reply:
Yeah I was thinking of that as well - but I recalled a conversation around the es6ification days where the prevailing opinion was all exported names should aim to be globally descriptive, i.e. shouldn't depend on the namespace of the module name etc.

Reply:
That said, I know that we do it this way for the messenger modules and it looks clean to me. One wrinkle might be that if a module is imported and aliased by lots of other modules, then it becomes a bit messy aliasing it all over the place and making sure it is aliased consistently, but I agree that it looks a bit odd having `initHighMerch` in the `high-merch.js` module as opposed to just `init`, it's like "well, yeah, duh".

Reply:
😭 that is absurd and contradicts decades of computer science. anyway, if that's the rule then forget what I said.

Reply:
yeah well, naming is difficult enough that it should be kept _local_ as much as possible. imagine if countries imposed the same constraint on children's first names! you would be jon45793289, nobody would be able to pronounce your name without a glass of water, but at least it would be unique.

Reply:
No! I agree! Ammending commit incoming....